crates/rooch/src/commands/account/commands/list.rs

```diff
@@ -69,9 +69,11 @@ impl CommandAction<Option<AccountsView>> for ListCommand {
         let password = if context.keystore.get_if_password_is_empty() {
             None
         } else {
-            Some(
+            let password = Some(
                 prompt_password("Enter the password to create a new key pair:").unwrap_or_default(),
-            )
+            );
+            println!();
```

Comment:
Please also help to change the `WalletContextOptions` prompt_password.

https://github.com/rooch-network/rooch/blob/8a16bd56a71a01c07677cd593aff602688619430/crates/rooch/src/cli_types.rs#L117-L135